File: Examen/Fortran/TablaDeMultiplicar.f
      PROGRAM TablaMultiplicar
      INTEGER N, I, RESULTADO

      PRINT *, 'Ingrese un numero para generar su tabla de multiplicar:'
      READ *, N

      PRINT *, 'Tabla de multiplicar de ', N
      DO 10 I = 1, 10
         RESULTADO = N * I
         PRINT *, N, ' x ', I, ' = ', RESULTADO
10    CONTINUE

      PRINT *, 'Presione ENTER para salir...'
      READ(*,*)
      END

File: Examen/C/TablaDeMultiplicar.c
#include <stdio.h>

int main() {
    int n, i, resultado;

    printf("Ingrese un numero para generar su tabla de multiplicar: ");
    scanf("%d", &n);

    printf("Tabla de multiplicar de %d:\n", n);
    for(i = 1; i <= 10; i++) {
        resultado = n * i;
        printf("%d x %d = %d\n", n, i, resultado);
    }

    return 0;
}
